.idea/vcs.xml

```diff
@@ -2,5 +2,9 @@
 <project version="4">
   <component name="VcsDirectoryMappings">
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/vendor/cotire" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/vendor/glfw" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/vendor/google/googletest" vcs="Git" />
```

Comment:
There must be some other method you use for adding submodule, because rest of us doesn't have these in vcs.xml. That's probably the reason you can push to these repositories from CLion. I would delete all of these lines.